onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2092,9 +2096,9 @@ void RegisterContribSchemas() {
                 .Add("StdDev = Sqrt (VarPlusEpsilon)")
                 .Add("Deviation = Sub (XU, Mean2D)")
                 .Add("Normalized = Div (Deviation, StdDev)")
```

Comment:
>Normalized [](http://example.com/codeflow?start=22&length=10)

This is a similar issue with the previous comment. 

Normalized: U, NormalizedV: V, Scale2D: V

When U = float, V = half, I think the following computation is not safe. 
Normalized in U --> Normalized in V --> Scaled in V

In our kernel registration, type U > V, U > T, unless we changed that,  otherwise we can reorder the compute to fix this problem.

Reply:
I think this is same logic as the sub-graph we want to fuse. The Cast Op is added right before the final Mul and Add from float to half. As said, the internal logic is on U for precision, then when it goes to the end, to produce V output, we cast it back to V and then do final calculation. One way is to cast both scale and bias to U, calculate and then cast back to V for output, but this is not necessary as we add more Cast Ops.